### DISCUSSION TITLE
Add diagram within text note

### DISCUSSION THREAD

**Discussion by MarcoFaccani** [TriliumNext/Trilium#2755] [Q&A] [answered]
Hi,
I couldn't figure out how to add a diagram to a text note. 
Example:

Some text...
     [Mermaid diagram]
...some more text

Is it currently possible?

**Comment by sigaloid** [comment, marked as answer]
No, mermaid notes are a separate type of note. However, you can create a child note of type "mermaid diagram" and add it to the parent note, meaning it should be visible from that note.

**Reply by MarcoFaccani** [reply]
Thank you Sigaloid, the result is anyhow exactly what I was looking for!

**Comment by zadam** [comment]
Another option is to use "include note": https://github.com/zadam/trilium/wiki/Text-notes
